tests/perf-regression-latency-in-memory.yaml

```diff
@@ -0,0 +1,74 @@
+test_duration: 300
+
+prepare_write_cmd: ["cassandra-stress write no-warmup cl=ALL n=9000000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..9000000",
+                    "cassandra-stress write no-warmup cl=ALL n=9000000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=9000001..18000000",
+                    "cassandra-stress write no-warmup cl=ALL n=9000000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=18000001..27000000",
+                    "cassandra-stress write no-warmup cl=ALL n=9000000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=27000001..36000000"]
+
+stress_cmd_r: "cassandra-stress read no-warmup  cl=ONE duration=50m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate 'threads=10 throttle=30000/s' -col 'size=FIXED(1024) n=FIXED(1)' -pop 'seq=1..36000000'"
+
+n_db_nodes: 3
+n_loaders: 4
+n_monitor_nodes: 1
+
+failure_post_behavior: 'destroy'
+round_robin: 'true'
+space_node_threshold: 644245094
+
+user_prefix: 'perf-regression-latency-in-memory-VERSION'
+ami_id_db_scylla_desc: 'VERSION_DESC'
+experimental: 'true'
+
+# 80GB in-memory storage - 36GB table is 45% of total in-memory store
+append_scylla_args: "--in-memory-storage-size-mb 80000"
+
+
+# email definitions to send regression report
+send_email: true
+email_recipients: ['roy@scylladb.com', 'shlomi@scylladb.com', 'bentsi@scylladb.com']
+
+backends: !mux
+    gce: !mux
+        cluster_backend: 'gce'
+        user_credentials_path: '~/.ssh/scylla-test'
+        gce_user_credentials: '~/Scylla-c41b78923a54.json'
+        gce_service_account_email: 'skilled-adapter-452@appspot.gserviceaccount.com'
+        gce_project: 'skilled-adapter-452'
+        gce_image: 'https://www.googleapis.com/compute/v1/projects/centos-cloud/global/images/family/centos-7'
+        gce_image_username: 'scylla-test'
+        gce_instance_type_db: 'n1-highmem-16'
+        gce_root_disk_type_db: 'pd-ssd'
+        gce_root_disk_size_db: 50
+        gce_n_local_ssd_disk_db: 1
+        gce_instance_type_loader: 'n1-standard-2'
+        gce_root_disk_type_loader: 'pd-standard'
+        gce_n_local_ssd_disk_loader: 0
+        gce_instance_type_monitor: 'n1-standard-2'
+        gce_root_disk_type_monitor: 'pd-standard'
+        gce_root_disk_size_monitor: 50
+        gce_n_local_ssd_disk_monitor: 0
+        scylla_repo: 'REPO_FILE_PATH'
+        us_east_1:
+          gce_datacenter: 'us-east1-b'
+
+    aws: !mux
+        user_credentials_path: '~/.ssh/scylla-qa-ec2'
+        cluster_backend: 'aws'
+        instance_provision: 'spot_low_price' # Allowed values: on_demand|spot_fleet|spot_low_price|spot_duration
```

Comment:
better to use spot_duration, it shall make sure that we get the nodes for the entire test duration.